public/coffee/ide/editor/directives/aceEditor/auto-complete/SnippetManager.coffee

```diff
@@ -1,12 +1,83 @@
 define () ->
+	noArgumentCommands = [
```

Comment:
Hard to tell from Github, but I think it's indented with a tab, while the rest of the file is spaces?